### ISSUE TITLE
No xml Generated 

### ISSUE THREAD

**Issue by GamerClassN7** [romainsi/zabbix-VEEAM_B-R#43]
Zabbix server and agent are 4.0.13
Veeam 9.5

not even with:
`PS C:\Windows\system32> powershell.exe -NoProfile -ExecutionPolicy Bypass -File "C:\Program Files\Zabbix Agent\scripts\z
abbix_vbr_job.ps1" ExportXml`

![image](https://user-images.githubusercontent.com/22167469/67933126-7fa29000-fbc5-11e9-84fa-405300ddc7ae.png)

![image](https://user-images.githubusercontent.com/22167469/67933180-98ab4100-fbc5-11e9-82e7-385fef116bec.png)

there are no xml Files only folder is created can anybody help me ?

**Comment by GamerClassN7** [comment]
I will try that tomorow

**Comment by GamerClassN7** [comment]
Now are xmls here :/ but for some reason in zabbix i still see only:
![image](https://user-images.githubusercontent.com/22167469/68196403-df7bab00-ffb8-11e9-893a-1c6fd7142106.png)

**Comment by GamerClassN7** [comment]
Could it be because of Timeout ? and How can i check that ??

**Comment by GamerClassN7** [comment]
Can i somehow run data transfer maualy to zabbix ?

**Comment by GamerClassN7** [comment]
![image](https://user-images.githubusercontent.com/22167469/68284239-65612a00-007d-11ea-884e-acd01d385160.png)

**Comment by GamerClassN7** [comment]
Any idea why ?

**Comment by romainsi** [comment]
Hi, 

Can you check your zabbix_agentd.conf contains the userparameter ?
`UserParameter=vbr[*],powershell -NoProfile -ExecutionPolicy Bypass -File "C:\Program Files\Zabbix Agent\scripts\zabbix_vbr_job.ps1" "$1" "$2" "$3"
`
On your zabbix server you can test that and return the error : 
`zabbix-get -s "yourveeamserver" -k 'vbr["DiscoveryBackupJobs"]'
`

**Comment by GamerClassN7** [comment]
First one yes
and second i will test that immediately

**Comment by GamerClassN7** [comment]
![image](https://user-images.githubusercontent.com/22167469/69789049-0dbb6780-11c0-11ea-90d4-cb8f8ed213ce.png)

**Comment by GamerClassN7** [comment]
Hello, Any news ?

**Comment by GamerClassN7** [comment]
Progress is that xml are generated but items are labeled in discover like unsupported even when i try it on machine 
![image](https://user-images.githubusercontent.com/22167469/86367103-dc39db80-bc7b-11ea-987f-35e0b2ac0de6.png)

**Comment by Randy29800** [comment]
Hi
Did you solve your problem?
I ve got the same :(

**Comment by GamerClassN7** [comment]
@Randy29800  yes, try to set Agents to active mod instead of passive  this helped me
